src/modules/displayList.js

```diff
@@ -0,0 +1,119 @@
+import completeIcon from '../images/completed.png';
+import checkBox from '../images/checkbox.png';
+import dots from '../images/threeDots.png';
+
+import trashIcon from '../images/trash.png';
+
+let tasks = [];
+const setLocalStorage = () => {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+};
+
+const listContainer = document.querySelector('.task-container');
+
+const displayList = () => {
+  listContainer.innerHTML = '';
+  tasks.forEach((task, index) => {
+    let checkTask;
+    let crossed;
+    if (task.completed) {
+      checkTask = completeIcon;
+      crossed = 'crossed';
+    } else {
+      checkTask = checkBox;
+    }
+    task.index = index;
+    const list = document.createElement('li');
+    list.classList.add('list');
+    list.innerHTML = ` <img class='checkbox' src='${checkTask}'> <p class='task-description ${crossed}'>${task.description}</p> <img class='options' src='${dots}'> `;
+    listContainer.appendChild(list);
+  });
+};
+displayList();
+
+const localTasks = localStorage.getItem('tasks');
+const getLocalStorage = () => {
+  if (localTasks) {
+    tasks = JSON.parse(localTasks);
+    displayList();
+  }
+};
+getLocalStorage();
+
+const submitBtn = document.querySelector('#submitBtn');
+const addInput = document.querySelector('.add');
+const addList = () => {
+  submitBtn.addEventListener('click', (e) => {
+    e.preventDefault();
+    const newTask = {
+      description: addInput.value,
+      completed: false,
+      index: tasks.length,
+    };
+    tasks.push(newTask);
+    setLocalStorage();
+    displayList();
+  });
```

Comment:
There are some issues that stop your app from running as expected. 👀

The functionality of the application is fluctuating. 🙄

Please kindly ensure that all functions of the app are working fine as expected so that users can "Update the value of the item" for a completed task and "Clear all completed tasks". 💪 🛠